marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -323,7 +331,15 @@ const Flows: React.FC<Props> = (props) => {
             onCancel={() => onCancel()}
             width={350}
         >
-            <div style={{fontSize: '16px', padding: '10px'}}>Are you sure you want to add step "{stepName}" to flow "{flowName}"?</div>
+            <div style={{fontSize: '16px', padding: '10px'}}>
```

Comment:
Can we move these styles into a CSS class? You may need to apply the class via flows.scss (instead of flows.module.scss) in order to override what's applied by Ant Design.

Reply:
Made a new style in flows.modules.scss.